io.github.mahmoudbahaa.outlook_for_linux.appdata.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+	<id>io.github.mahmoudbahaa.outlook_for_linux</id>
+	<metadata_license>CC0-1.0</metadata_license>
+	<project_license>GPL-3.0</project_license>
+	<name>outlook-for-linux</name>
+	<summary>Unofficial Microsoft Outlook client for Linux using Electron</summary>
+	<description>
+		<p>Unofficial Microsoft Outlook client for Linux using Electron.</p>
+		<p>It uses the Web App and wraps it as a standalone application using Electron.</p>
+		<p>A clone of <b>teams-for-linux</b> but for outlook. All credits for authors of that repo</p>
+	</description>
+	<url type="homepage">https://github.com/mahmoudbahaa/outlook-for-linux</url>
+	<url type="help">https://github.com/mahmoudbahaa/outlook-for-linux/blob/develop-outlook/KNOWN_ISSUES.md</url>
+	<url type="bugtracker">https://github.com/mahmoudbahaa/outlook-for-linux/issues</url>
+	<launchable type="desktop-id">io.github.mahmoudbahaa.outlook_for_linux.desktop</launchable>
+	<releases>
+		<release version="1.3.13" date="2023-10-17">
+			<description>
+				<p> Initial release same as teams-for-linux 1.3.13 but for outlook</p>
+				<p> See https://github.com/IsmaelMartinez/teams-for-linux for actual changelog </p>
+			</description>
+		</release>
+	</releases>
+	<developer_name>Mahmoud Bahaa</developer_name>
+	<screenshots>
+		<screenshot type="default">
+			<image type="source">https://iili.io/sUyAc7.png</image>
```

Comment:
Host this somewhere else, more reliable like in your upstream repo

Reply:
done